src/style/search/search-bar.less

```diff
@@ -29,10 +29,11 @@
     background: no-repeat left @bg-panel;
     background-size: @text-xxlarge;
     background-position: 1.7rem 1.5rem;
-    top: 25%;
-    left: 50%;
+    top: round(25%);
```

Comment:
This looks suspicious to me. I would guess that this is equivalent to `25%`, but I'm not sure. Registering this concern for posterity.